components/microsoft_onedrive/sources/new-file-in-folder/new-file-in-folder.mjs

```diff
@@ -0,0 +1,41 @@
+import onedrive from "../../microsoft_onedrive.app.mjs";
+
+import {
+  hooks,
+  props,
+  methods,
+  run,
+} from "../common/base.mjs";
+
+import { toSingleLineString } from "../common/utils.mjs";
+
+export default {
+  type: "source",
+  key: "microsoft_onedrive-new-file-in-folder",
+  name: "New File in Folder (Instant)",
+  description: toSingleLineString(`
+    Emit an event when a new file is added to a
+    specific directory tree in a OneDrive drive
+  `),
```

Comment:
There needs to be a space between "a" and "specific". In the UI this shows up as "aspecific".